content/docs/user-guide/action-manager/fundamentals/concepts/actions.md

```diff
@@ -0,0 +1,295 @@
+---
+title: Actions
+linkTitle: Actions
+description: 
+weight: 101
+---
+
+The Action Manager system provides interfaces to register actions and related frameworks to interact with the O3DE Editor.
+
+
+## Action Context
+
+An Action Context is an object that owns a collection of Actions.
+
+An Action Context is comprised of:
+
+* A string **identifier** that makes it possible to reference it in API calls;
+* A **name** string, which is meant to be a human-readable description to show in UI (currently unused);
+
+
+### Registration
+
+It is possible to register a new Action Context via the following API call:
+
+```
+AzToolsFramework::ActionContextProperties contextProperties;
+contextProperties.m_name = "Test Action Context";
+
+actionManagerInterface->RegisterActionContext(
+    "o3de.context.identifier", 
+    contextProperties
+);
+    
+```
+
+### Associating Widgets
+
+The purspose of Action Contexts is to bind the group of actions it owns to a widget in the Editor so that they can be accessed via keyboard shortcuts contextually to the user focus.
+See the [Hotkeys]() page for more information.
+
+
+## Action
+
+An Action is an abstraction of a re-usable Editor interaction. It is not accessible upon creation, but becomes available to the user once it's added to a menu or toolbar, or if a hotkey is assigned to it.
+
+An Action is comprised of
+
+* A string **identifier** that makes it possible to reference it in API calls;
+* A string **context identifier** that defines the Action Context that owns this Action;
+* A behavior in the form of a function to be called whenever the Action is triggered.
+
+
+### Registration
+
+An action can be registered via the following API call:
+
+```
+actionManagerInterface->RegisterAction(
+    "o3de.context.identifier",
+    "o3de.action.identifier",
+    {},
+    []
+    {
+        /* Trigger Behavior */
+    }
+);
+```
+
+The behavior can either be provided as a lambda or a function reference.
+
+Additional descriptors and decorators can be added as Action Properties:
+
+* A **name** string, which succinctly describes the effect of the action and is used in UI like Menus or ToolBars;
+* A **description** string, which can be used to provide additional information to the user, usually shown in a tooltip when hovering over the action in a UI;
+* A **category** string, which is meant to be used to categorize similar strings in the UI (currently unused);
+* An **icon path** pointing to an svg image that will be used to represent the action in Menus and ToolBars. It needs to be a Qt path for an svg file defined in a qrc resource file.
```

Comment:
Same here, preference to remove "currently unused", if it should be connected on the code-side
```suggestion
* A **name** string, which succinctly describes the effect of the action and is used in UI like menus or toolbars;
* A **description** string, which can be used to provide additional information to the user, usually shown in a tooltip when hovering over the action in a UI;
* A **category** string, which is meant to be used to categorize similar strings in the UI;
* An **icon path** pointing to an svg image that will be used to represent the action in Menus and toolbars. It needs to be a Qt path for an svg file defined in a qrc resource file.
```